roles/StackStorm.rabbitmq/tasks/rabbitmq_redhat.yml

```diff
@@ -0,0 +1,59 @@
+---
+
+- name: RabbitMQ on RHEL8 requires module(perl:5.26)
```

Comment:
Considering we moved to official RabbitMQ, is this requirement still a thing?
I see it's from the previous Ansible role.

Reply:
@armab I took it out and install was fine, so no-longer required.